### ISSUE TITLE
Dubious error message when inline initializing a slice

### ISSUE THREAD

**Issue by asahi3g** [skycoin/cx#105]
```
package main
func main()() {
    var ai []i32 = []i32{0, 1, 2, 3, 4}
}
```
output:
```
error: pending_0.cx:4 operator 'identity' expects to return 1 output, but 2 receiving arguments were provided
```

**Comment by corpusc** [comment]
again, probably irrelevant to you...

....but assigning values on the same line with var declaration is very iffy.
separating them on multiple lines seems reliable.

**Comment by amherag** [comment]
This was fixed in v0.6.0.
